zzz_generated/app-common/clusters/AccessControl/CommandIds.h

```diff
@@ -0,0 +1,24 @@
+// DO NOT EDIT MANUALLY - Generated file
+//
+// Identifier constant values for cluster AccessControl (cluster code: 31/0x1F)
+// based on src/controller/data_model/controller-clusters.matter
+#pragma once
+
+#include <lib/core/DataModelTypes.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace AccessControl {
+namespace Commands {
+namespace ReviewFabricRestrictions {
+inline constexpr CommandId Id = 0x00000000;
+} // namespace ReviewFabricRestrictions
+namespace ReviewFabricRestrictionsResponse {
```

Comment:
This and the attribute/event ids would be a lot more readable with some blank lines.... followup is fine.